### PR TITLE
Feature/migrate front end code

### PR DESCRIPTION
**Pull request by Iyadbitar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,49 @@ go generate ./ui
...
+# Red Sky Frontend
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```
Comment (Ofer-Idan) [inline README.md]: This is all boilerplate README from react, right?

**Comment by Ofer-Idan** [comment]
Looks good to me. Two things I'd like to look at:
1. The UI README is just boilerplate - I'd prefer to rewrite it so that it's actually usable (e.g. there's no mention of "npm install" which needs to happen before you can run).
2. Running the UI, it looks like it's pulling an outdated experiment list - is it pulling from a static list at the moment? It looked as though it should be pinging the API directly already.

File: README.md (hunk 1 above)
Comment (Iyadbitar) [reply README.md]: I updated the readme file

**Comment by Iyadbitar** [comment]
1. the readme file is updated 
2. yes the data is stubbed and the app is not calling the server api yet as far as my understanding the data returned is based on the API key and secrete and the ones I have so far returning the same data.

**Pull request merged**